src/test/java/org/heigit/ohsome/ohsomeapi/controller/GetControllerTest.java

```diff
@@ -84,15 +82,15 @@ public void getMetadataTest() {
   public void getGeneralResourceWithFalseParameterTest() {
     TestRestTemplate restTemplate = new TestRestTemplate();
     ResponseEntity<JsonNode> response =
-        restTemplate.getForEntity(server + port + "/elements/area?type=way", JsonNode.class);
+        restTemplate.getForEntity(server + port + "/elements/area?filterr=type:way", JsonNode.class);
```

Comment:
this line is now longer than 100 characters :see_no_evil: 

https://jenkins.ohsome.org/job/ohsome-api/view/change-requests/job/PR-100/14/checkstyle/fileName.1150936862/source.1421a870-0b11-43d6-8848-f23fb0c71e35/#85